themes/godotengine/layouts/features.htm

```diff
@@ -55,12 +55,12 @@ <h2>innovative design</h2>
       <p>
         <em>Create games with ease, using Godot's unique approach to game development.</em>
         <ul class="text-left">
-          <li><b>There is a node for that</b>, godot comes with hundreds of built-in nodes that make game design a breeze, you can also create you own, with custom behaviors, editors and much more.</li>
-          <li><b>Amazing scene system</b>, create node compositions with support for instancing and inheritance.</li>
-          <li><b>Visual editor</b>, with all the tools you need packed into a beautiful and uncluttered context-sensitive UI.</li>
+          <li><b>There is a node for that.</b> Godot comes with hundreds of built-in nodes that make game design a breeze. You can also create your own for custom behaviors, editors and much more.</li>
```

Comment:
This "There is a node for that." comes a bit out of the blue IMO, I don't think it's really understandable. It sounds like the answer to the unformulated question "How to implement <thing>?", but since it's not formulated, it's weird :)

"There is a node for all your needs." might be better, while staying close to the original intention.